bin/sync_slurm_acct.py

```diff
@@ -108,12 +102,17 @@ def main():
             clusters = VSC_SLURM_SYNC_CLUSTERS[host_institute]
         sacctmgr_commands = []
 
-        # make sure the institutes and the default accounts (VOs) are there for each cluster
-        sacctmgr_commands += slurm_institute_accounts(slurm_account_info, clusters, host_institute)
-
         # All users belong to a VO, so fetching the VOs is necessary/
         account_page_vos = [mkVo(v) for v in client.vo.institute[opts.options.host_institute].get()[1]]
 
+        # make sure the institutes and the default accounts (VOs) are there for each cluster
+        institute_vos = dict([
+            (v.vsc_id, v)
+            for v in account_page_vos
+            if v.vsc_id in INSTITUTE_VOS_BY_INSTITUTE[host_institute].values()
```

Comment:
does this work in py3 (`.values` is an iterator, probably doing some dumb things with `in`)

Reply:
Works fine. `values()` is actually not an iterator in Python 3 , it's something else:

```
>>> foo = {'one': 1, 'two': 2}
>>> 'one' in foo.values()
False
>>> 1 in foo.values()
True
>>> bar = foo.values()
>>> next(bar)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'dict_values' object is not an iterator
```

Reply:
@boegel thanks for checking this. py3 weirdness...

Reply:
@boegel thanks.